raft/src/main/java/org/apache/kafka/raft/KafkaNetworkChannel.java

```diff
@@ -187,6 +189,8 @@ static AbstractRequest.Builder<? extends AbstractRequest> buildRequest(ApiMessag
             return new FetchRequest.SimpleBuilder((FetchRequestData) requestData);
         if (requestData instanceof FetchSnapshotRequestData)
             return new FetchSnapshotRequest.Builder((FetchSnapshotRequestData) requestData);
+        if (requestData instanceof UpdateRaftVoterRequestData)
```

Comment:
I believe `RemoveRaftVoterRequestData` needs to be added as well.

Reply:
Do the controllers send `RemoveRaftVoterRequestData` and `AddRaftVoterRequestData` here? I don't think so?

Reply:
that's what I thought as well, I don't think we need to add either here

Reply:
FWIW, I don't see any errors when I use `remove-controller` so it doesn't look like we need to support this.